apps/learning-snippets/src/test/property-grid/Usage.test.tsx

```diff
@@ -0,0 +1,100 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/* eslint-disable import/no-duplicates */
+import { expect } from "chai";
+import sinon from "sinon";
+import { StagePanelLocation, StagePanelSection, StageUsage,  } from "@itwin/appui-react";
+import { PresentationPropertyDataProvider } from "@itwin/presentation-components";
+// __PUBLISH_EXTRACT_START__ PropertyGrid.RegisterPropertyGridWidgetImports
+import { createPropertyGrid } from "@itwin/property-grid-react";
+import { UiItemsManager } from "@itwin/appui-react";
+// __PUBLISH_EXTRACT_END__
+// __PUBLISH_EXTRACT_START__ PropertyGrid.RegisterCustomPropertyGridWidgetImports
+import {
+  AddFavoritePropertyContextMenuItem,
+  AncestorsNavigationControls,
+  CopyPropertyTextContextMenuItem,
+  IModelAppUserPreferencesStorage,
+  RemoveFavoritePropertyContextMenuItem,
+  ShowHideNullValuesSettingsMenuItem,
+} from "@itwin/property-grid-react";
+import type { IModelConnection } from "@itwin/core-frontend";
+// __PUBLISH_EXTRACT_END__
+import { PropertyGridTestUtils } from "../../utils/PropertyGridTestUtils";
+
+describe("Property grid", () => {
+  describe("Learning snippets", () => {
+    describe("Usage", () => {
+      beforeEach(async () => {
+        await PropertyGridTestUtils.initialize();
+      });
+
+      afterEach(async () => {
+        await PropertyGridTestUtils.terminate();
+        sinon.restore();
+        // eslint-disable-next-line @itwin/no-internal
+        UiItemsManager.clearAllProviders();
+      });
+
+      it("Registers property grid", async function () {
+        // __PUBLISH_EXTRACT_START__ PropertyGrid.RegisterPropertyGridWidget
+        UiItemsManager.register({ id: "property-grid-provider", getWidgets: () => [createPropertyGrid({})] });
+        // __PUBLISH_EXTRACT_END__
+
+        expect(UiItemsManager.getWidgets?.("", StageUsage.General, StagePanelLocation.Right, StagePanelSection.End)).to.not.be.empty;
+        expect(UiItemsManager.getWidgets?.("", StageUsage.General, StagePanelLocation.Right, StagePanelSection.Start)).to.be.empty;
+        expect(UiItemsManager.getWidgets?.("", StageUsage.General, StagePanelLocation.Left, StagePanelSection.Start)).to.be.empty;
```

Comment:
I think one check is enough. We don't need to test AppUI

```suggestion
        expect(UiItemsManager.getWidgets?.("", StageUsage.General, StagePanelLocation.Right, StagePanelSection.End)).to.not.be.empty;
```